### DISCUSSION TITLE
New argument or flag for only pulling numbered versions

### DISCUSSION THREAD

**Discussion by Ted-Corrrell** [containrrr/watchtower#1178] [Feature Requests] [answered]
It would be great to have an option to pull the latest numbered version of a container, instead of latest. I have had issues before where I had to roll back to an older version due to a bug, but even though I had the older image still, I had a hard time using it, because after it was no longer :"latest" I couldn't specify what version it was. Since then I have been downloading the latest versions, by version number instead of by simply using "latest". Being able to let watchtower take care of my updates again would help me stay current with all my containers.

**Comment by simonaronsson** [comment, marked as answer]
Following semvers is something that has been discussed at length previously, and the conclusion was that we (as in me and @piksel) won't be pursuing that anytime soon. If someone would like to have a stab at it, they're more than welcome to, and we'd do our best to help them through the process of getting it reviewed, tested and, eventually, merged.

**Reply by Ted-Corrrell** [reply]
That is understandable. Unfortunately I would need to pick up a couple new skills in order to not break everything, and I don't have the time for that right now.
